README.md

```diff
@@ -93,3 +93,5 @@ The required resources for Google Cloud are detailed in the `src/gcp.tf` file. T
 ## Improvements
 
 In the future, we could make things better by splitting the settings for different cloud services like Azure, AWS, and Google Cloud into their own separate parts. This would make it easier and more flexible to work with each one on its own. It would help users handle their settings for each cloud service by themselves. This way, if you're just working with one cloud service, things would be smoother.
+
+For now you have to comment out the code in `aws.tf`, `azure.tf`, or `gcp.tf` and the corresponding outputs in `outputs.tf` for the cloud providers you do not want to use.
```

Comment:
Indeed. I suppose using modules or a `count` could make the experience better.

Reply:
I think modules make the most sense!